src/niscope/templates/session.py/fetch_waveform.py.mako

```diff
@@ -15,13 +15,13 @@
         num_samples = int(len(wfm) / self._actual_num_wfms())
 
         if wfm.dtype == numpy.float64:
-            return self._fetch_into(num_samples=num_samples, wfm=wfm, timeout=timeout)
+            return self._fetch_into_numpy(num_samples=num_samples, wfm=wfm, timeout=timeout)
         elif wfm.dtype == numpy.int8:
-            return self._fetch_binary8_into(num_samples=num_samples, wfm=wfm, timeout=timeout)
+            return self._fetch_binary8_into_numpy(num_samples=num_samples, wfm=wfm, timeout=timeout)
```

Comment:
why rename these?
won't they work with array.array anymore?

Reply:
They don't work with `array.array` now.

I renamed these because I wanted to add `array.array` but saw it was going to be more involved that I wanted for this PR. I decided to leave the '_numpy' suffix because I would like to get '_into_array' supported but in a different PR.